app/views/DR/HomeScreen/index.js

```diff
@@ -164,6 +165,46 @@ class HomeScreen extends Component {
       state: { confirmed, deaths, recovered, current, refreshing, notified },
     } = this;
 
+    const dashboardCards = () => {
+      const dashboardCategories = [
+        { label: 'label.positive_label', styles: '', value: confirmed },
+        {
+          label: 'label.deceased_label',
+          styles: { color: Colors.BUTTON_LIGHT_TEX },
+          value: deaths,
+        },
+        {
+          label: 'label.recovered_label',
+          styles: { color: Colors.GREEN },
+          value: recovered,
+        },
+        {
+          label: 'label.case_day_label',
+          styles: { color: Colors.SUN },
+          value: current,
+        },
+      ];
+      const CONFIRMED_COVID_CASES_URL =
+        'https://coronavirusrd.gob.do/casos-de-covid-19-confirmados/';
+
+      return dashboardCategories.map(obj => {
+        return (
+          <TouchableOpacity
+            style={styles.shadowBorder}
+            key={Index}
+            onPress={() =>
+              navigation.navigate('Details', {
+                source: { uri: CONFIRMED_COVID_CASES_URL },
+              })
+            }>
+            <Card style={styles.infoCards}>
+              <Text style={[styles.dataText, obj.styles]}>{obj.value}</Text>
+              <Text style={styles.text}>{t(obj.label)}</Text>
+            </Card>
+          </TouchableOpacity>
+        );
+      });
+    };
```

Comment:
I think it would be cleaner and better to read if you move this piece of code to other file